sql/Cargo.toml

```diff
@@ -27,9 +27,9 @@ datafusion-expr = { workspace = true }
 datafusion-proto = { workspace = true }
 df_operator = { workspace = true }
 hashbrown = { version = "0.12", features = ["raw"] }
-influxql-logical-planner = { git = "https://github.com/CeresDB/influxql", package = "iox_query_influxql" }
-influxql-parser = { git = "https://github.com/CeresDB/influxql", package = "influxdb_influxql_parser" }
-influxql-schema = { git = "https://github.com/CeresDB/influxql", package = "schema" }
+influxql-logical-planner = { git = "https://github.com/jiacai2050/influxdb_iox", branch = "ceresdb", package = "iox_query_influxql" }
```

Comment:
For simplicity, I use iox directly. will move to influxql crates when all influxql features are supported.

Reply:
OK